src/main/java/com/softserveinc/ita/pageobjects/ProductModel.java

```diff
@@ -0,0 +1,59 @@
+package com.softserveinc.ita.pageobjects;
+
+import com.codeborne.selenide.SelenideElement;
+import io.qameta.allure.Step;
+
+import java.util.List;
+
+import static com.codeborne.selenide.CollectionCondition.sizeGreaterThan;
+import static com.codeborne.selenide.Condition.visible;
+import static com.codeborne.selenide.Selenide.$$x;
+import static com.codeborne.selenide.Selenide.$x;
+import static java.time.Duration.ofSeconds;
+import static java.util.stream.Collectors.toList;
+
+public class ProductModel extends BasePage<ProductModel> {
```

Comment:
how it's could be a model? the model should represent an object, but not a page object

also, I think all things should be in SearchResultPage

Reply:
Probably, then we dont need a productmodel...

Reply:
I tought that ProductModel means some product construction with some functions that attach to any random product